docs/pages/api.rst

```diff
@@ -435,6 +447,11 @@ Interface: `Containsable`_
 
 Signature: ``Collection::contains(...$value);``
 
+.. code-block:: php
+
+    $collection = Collection::fromIterable(range('a', 'c'))
+        ->contains('d'); // [false]
```

Comment:
this behaviour is based on the spec but I have to say it surprised me. Shouldn't the return type be just `bool`? I would expect to use this method as such:

```PHP

$collection = Collection::fromIterable([1, 2, 3]);

if ($collection->contains(2)) {
    // do something
}
```

How is the current implementation meant to be used?

Reply:
Indeed, this was a choice that I made in order to make sure that the collection would always return a collection.
The 3 only methods that are not potentially returning a Collection are: `all`, `key`, `current`.
In this example, to get the boolean, you have to do:

```php
$collection = Collection::fromIterable([1, 2, 3]);

if ($collection->contains(2)->current()) {
    // do something
}
```

Reply:
that makes sense, I added a note [here](https://github.com/loophp/collection/pull/84/commits/21bdbe8bf781d00c5078dcde232e1ae3c48b65e7#diff-89f6324da13155a697ae0b42136e03a40892dd4edc4257ea1e451f0ea2cc3f80R148-R150) to highlight this a bit better and added to the [contains example](https://github.com/loophp/collection/pull/84/commits/21bdbe8bf781d00c5078dcde232e1ae3c48b65e7#diff-89f6324da13155a697ae0b42136e03a40892dd4edc4257ea1e451f0ea2cc3f80R457-R459)